src/components/CEnqueueForm.vue

```diff
@@ -42,7 +41,7 @@
             class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
             for="grid-last-name"
           >
-            delay(min)
+            delay(ms)
```

Comment:
is it me or this value is not used on the backend?

Reply:
This value is used on the backend. Be careful : as is shown in the changes, the value is in milliseconds and not in minutes, so if you make a test with a value such as 1 you might not see the delay.

Reply:
indeed. I think it should be part of the Options section